computer-vision/README.md

```diff
@@ -1,4 +1,34 @@
+
+## Tools and libraries to get you started
+If you're gonna do any kind of CV project, I'd highly recommend python. There is a lot of documentation/tutorials for common CV related libraries online. C++ and Java are also pretty popular options, but I can't speak to their usability (if you can, please contribute!).
+
+If you haven't been able to tell yet, the most popular CV library is [OpenCV](opencv.org). There's a whole bunch of documentation out there for this, I'll link a couple python tutorials I found really useful down below. The best part about this library is that it works right out of the box (for the most part), allowing you to get the video data you need quickly and move right on integrating other technologies. 
+
+Another super useful (python) package to check out is the [face-recognition](https://github.com/ageitgey/face_recognition) package on PyPI. Again, this module works out of the box, allowing you to get the data you need quickly. The easiest things to do using this library are finding/recognizing known faces. With a little practice and digging though, you can delve deeper and get different facial "landmarks" (Landmarks is just a fancy term referring to the various features on a face - eyes, nose, mouth, stuff like that). The best resource for using this library is the examples folder in their github. For hackathon purposes, its usually easiest to copy one of their examples and add your own modifications. 
+
+I built a cool little project using facial landmarks to detect if the person in front of the camera has fallen asleep ([drowsy-driver](https://github.com/kdesai2018/drowsy-driver)). Again, this project puts together a bunch of technologies - Google Maps API, arduino controlled hardware and the python VLC library for audio. 
+
+## Takeaways
+Computer Vision is a really useful tool for any hackathon project. However, it works best in conjuction with another technology, so finding a cool way to integrate it is key. One caveat, however, is that CV might not be necessary for your project, especially if there is existing data on the internet that can serve your needs. Good luck!
+
+## Resources I promised to link
+1. [Official OpenCV tutorial](https://opencv-python-tutroals.readthedocs.io/en/latest/py_tutorials/py_tutorials.html)
+2. [Walks you through how to set up environment - for absolute beginners](https://towardsdatascience.com/computer-vision-for-beginners-part-1-7cca775f58ef?gi=b8f07802223d)
+3. [one of the most useful blogs I've found about advanced OpenCV + Python operations](https://www.pyimagesearch.com). Navigate this site using the "Topics" dropdown. 
+
+These sites are just a starting point - there is no substitute for getting your hands messy quickly. Fire up a venv (or conda env) and get started! 
```

Comment:
kids won't know what a venv or conda env is, i barely figured it out lol. maybe write about that and like how to run a program